doc/connectivity/bluetooth/bluetooth-le-host.rst

```diff
@@ -153,6 +153,119 @@ an application comes in direct contact with it only when using it in the
 so-called Connection-oriented Channels (CoC) mode. More information on
 this can be found in the :ref:`L2CAP API section <bt_l2cap>`.
 
+Terminology
+-----------
+
+The definitions are from the Core Specification version 5.4, volume 3, part A
+1.4.
```

Comment:
```suggestion
The definitions are from the Core Specification version 5.4, volume 3, part A, section 1.4.
```

Also FYI the recommended line length in Zephyr is 100, which may improve some formatting in this PR

Reply:
Nice, we should use that for new files then.